components/emoji_picker/emoji_picker.jsx

```diff
@@ -472,15 +480,22 @@ export default class EmojiPicker extends React.PureComponent {
         return (
             <div className='emoji-picker__search-container'>
                 <span className='fa fa-search emoji-picker__search-icon'/>
-                <input
-                    aria-label={Utils.localizeMessage('emoji_picker.search_emoji', 'Search for an emoji')}
-                    ref={this.emojiSearchInput}
-                    className='emoji-picker__search'
-                    type='text'
-                    onChange={this.handleFilterChange}
-                    onKeyDown={this.handleKeyDown}
-                    placeholder={Utils.localizeMessage('emoji_picker.search', 'search')}
-                />
+                <FormattedMessage
+                    id='emoji_picker.search_emoji'
+                    defaultMessage='Search for an emoji'
+                >
+                    {(ariaLabel) => (
+                        <input
+                            aria-label={ariaLabel}
+                            ref={this.emojiSearchInput}
+                            className='emoji-picker__search'
+                            type='text'
+                            onChange={this.handleFilterChange}
+                            onKeyDown={this.handleKeyDown}
+                            placeholder={Utils.localizeMessage('emoji_picker.search', 'search')}
```

Comment:
I omitted it because the ticket didn't mention about the `placeholder` attribute but I can change it if you want :)

Reply:
Oh you're right.  I'm good with it as is though it will be great to change here so we can totally get rid off the `localizeMessage` or utils package for this component.

Also separate item, the default message should change from `'search'` to `'Search Emoji'` to match the one from [en.json](https://github.com/mattermost/mattermost-webapp/blob/8c5485094631c00f2385a9bf40dcde97d4b95bb0/i18n/en.json#L1780).

Reply:
@saturninoabril updated.